src/VisualStudio/Core/Def/Implementation/TableDataSource/AbstractRoslynTableDataSource.cs

```diff
@@ -59,8 +61,18 @@ private void OnSolutionCrawlerProgressChanged(object sender, ProgressData progre
 
         private void SolutionCrawlerProgressChanged(bool running)
```

Comment:
would really like some threading comments on these.

Reply:
would love some invariant checking.  i.e. can we change from 'running' to 'running'?

Reply:
some general docs/explanation woudl be good.

Reply:
It is doc'ed here: https://github.com/dotnet/roslyn/blob/5222059b8cddfa0de6432e0b1a2cd40327f30b58/src/Features/Core/Portable/SolutionCrawler/ISolutionCrawlerProgressReporter.cs#L21-L27